esmvalcore/cmor/_fixes/CMIP5/GFDL_ESM2G.py

```diff
@@ -37,7 +37,7 @@ def fix_metadata(self, cubes):
         return cubes
 
 
-class co2(Fix):
+class co2(Fix): # noqa
```

Comment:
Unrelated question, would it be possible to just use `class Co2(Fix)` etc. everywhere? Variable names are not case sensitive anyway and it would avoid a lot of linter errors that cause confusion for new contributors. I would be happy to make a pull request if you think it's too much work but do not object otherwise.

Reply:
Replied in #101